### ISSUE TITLE
Not able to properly use custom elements inside a controller in angularjs 

### ISSUE THREAD

**Issue by Aggelost1** [angular/angular.js#16693]
**I'm submitting a ...**

- [x] bug report
- [ ] feature request
- [ ] other 

**Current behavior:**
when I use an angular 6 custom element inside  an angularjs html **that has its own controller** (as part of the app works fine) then changes made in the custom element  are not detected (by the custom element). It is as if the ChangeDetector of the angular 6 component gets overridden.
**Expected / new behavior:**

I should be able to use custom elements no matter if they are in the main app or part of a controller. 

**Minimal reproduction of the problem with instructions:**

*STEPS TO REPRODUCE* : 
0: follow an angular6 custom element tutorial (I followed this https://www.telerik.com/blogs/getting-started-with-angular-elements and  and get the 5 files needed to use the custom element. the element should have a 2 way data binding (for me it is just an input and below this a div where I show the string of the input in real time).
1:  Add the files as scripts to a basic angularjs project as scripts at the end of ur html
2:  Observe that your custom element works fine if used  in the angularjs main scope
3:  Use it inside a controller, and observe that it renders ok but the changes made to it dont get registered by ur 2(or 1) way data binding.

*plunker:   http://plnkr.co/edit/yUanCgoqJO8Es1lsAIG4?p=preview
* how ever i could not upload my vendor.js file since it was too big but the angularjs project is there on the index.html
**AngularJS version:** 1.6.9
Check whether this is still an issue in the most recent stable or in the snapshot AngularJS
version (https://code.angularjs.org/snapshot/) : yes it is 

**Browser:** [all | Chrome XX | Firefox XX | Edge XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ]

I used chrome 68 (but browser should not be relevant i ve used multiple ones and the issue was in all)
on a Mac. high sierra 10.13.6 
**Anything else:**
there is a way to use the custom element if you make it detect changes on an interval i.e. `constructor( private ref: ChangeDetectorRef) {
        ref.detach();
        setInterval(() => {
             this.ref.detectChanges();
         }, 100);
    }` 
but this is way too "expensive". 

This is my 1st time reporting an issue if you think this should be raised in the angular 6 depo instead of here please let me know... 
attached is my vendor.js which i could not upload into plunker due to size
[vendor.js.zip](https://github.com/angular/angular.js/files/2380410/vendor.js.zip)

**Comment by gkalpak** [comment]
Excited to see you are trying to use an Angular Custom Element in an AngularJS app :champagne:

AngularJS does not do anything special with Custom Elements; let alone Angular change detection. I think this issue belongs in the [Angular repo][1].

[1]: https://github.com/angular/angular

**Comment by gkalpak** [comment]
Yeah, I took a look and there doesn't seem to be AngularJS related. (It is hard to be 100% sure without access to the sources, but that I'm 99% sure.)

**Comment by gkalpak** [comment]
> Yeah, I took a look and this doesn't seem to be AngularJS related. (It is hard to be 100% sure without access to the sources, but that I'm 99% sure 😁)
